doc/installation.rst

```diff
@@ -167,7 +161,7 @@ FFTW3
 If you have administrator access to the computers on which you use EOS,
 we recommend you install the above packages via your system's software management system.
 
-On a Debian/Ubuntu based operating system you can install the prerequisite software with the ``APT`` package management system,
+On a Debian/Ubuntu-based operating system you can install the prerequisite software with the ``APT`` package management system,
```

Comment:
You used ``apt`` above, while here we use ``APT``. Lowercase is probably better, since it matches the actual command.